akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -900,8 +900,9 @@ private[akka] class ActorSystemImpl(
     else {
       val message =
         if (isTypedGuardian)
-          s"cannot create top-level actor [$name] from the outside on a typed ActorSystem.  A classic ActorSystem " +
-          "should be created instead and used to spawn typed actors."
+          s"cannot create top-level actor [$name] from the outside on a typed ActorSystem.  In a typed ActorSystem, " +
```

Comment:
Suggest what to do if you want to keep a typed ActorSystem (spawn as a child of the guardian... maybe "spawned by the guardian behavior" is clearer?) then break the news that if that doesn't work, you need a classic ActorSystem.